Makefile

```diff
@@ -161,3 +116,26 @@ endif
 
 clean:
 	rm -rf *.created dist *.tar vendor
+
+run-plugin: run-etcd dist/libnetwork-plugin
+	-docker rm -f dind
+	docker run -h test --name dind --privileged -e ETCD_ENDPOINTS=http://$(LOCAL_IP_ENV):2379 -p 5375:2375 -d -v ${PWD}/dist/libnetwork-plugin:/libnetwork-plugin -ti docker:$(DOCKER_VERSION) --cluster-store=etcd://$(LOCAL_IP_ENV):2379
+	docker exec -tid --privileged dind /libnetwork-plugin
+	# To speak to this docker:
+	# export DOCKER_HOST=localhost:5375
+
+.PHONY: test
+# Run the unit tests.
+test: run-plugin
+	CGO_ENABLED=0 ginkgo 
+
+test-containerized: run-plugin
+# TODO - It would be nicer if this got the docker binary from the dind container
+	docker run --rm --net=host \
```

Comment:
As discussed: we need to visit the idea of having a centralized glide/build/test container that would make this incantation slightly less icky.    Can we have an issue to make sure we don't lose track of that?